### PR TITLE
Fix slack bug

### PR DESCRIPTION
**Pull request by rickducott**
BOT NOTES: 
resolves https://github.com/solo-io/go-utils/issues/315

### CODE REVIEW

**Comment by rickducott** [comment]
/kick

First flake was on `test_container_test`, second flake on `kubeinstall_test`

**Comment by rickducott** [comment]
Removing WIP because the libraries are sufficient for unit testing the bots now.

File: githubutils/repo_client.go [hunk 1]
```diff
@@ -158,3 +162,22 @@ func (c *repoClient) UpdateRelease(ctx context.Context, release *github.Reposito
...
+
+func (c *repoClient) CreateStatus(ctx context.Context, sha string, status *github.RepoStatus) (*github.RepoStatus, error) {
+	st, _, err := c.client.Repositories.CreateStatus(ctx, c.owner, c.repo, sha, status)
+	return st, err
+}
+
+func (c *repoClient) CreateComment(ctx context.Context, pr int, comment *github.IssueComment) error {
+	_, _, err := c.client.Issues.CreateComment(ctx, c.owner, c.repo, pr, comment)
+	return err
+}
```
Comment (mitchdraft) [inline githubutils/repo_client.go]: would be nice to add some tests here to round off the coverage

**Comment by mitchdraft** [comment]
tests look great!
/kick

**Pull request merged**